CONTRIBUTING.md

```diff
@@ -10,7 +10,8 @@ Install these dependencies:
 
 1. [Node](https://nodejs.org/en/download/package-manager/) - the version specified in the `.node-version` file at the root folder.
 2. [Yarn](https://yarnpkg.com/) - a package manager for installing dependencies and starting Brim in dev mode.
-3. [Go](https://go.dev/doc/install) - to compile some Zed dependencies.
+3. [Go](https://go.dev/doc/install) - to compile some [Zed](https://zed.brimdata.io/) dependencies.
+4. Typical command line tools, such as `make`, `unzip`, and `curl`
```

Comment:
I called these out because I found that on a minimal install of Ubuntu or Windows, some of these tools were not present. The debug logs generated when the `yarn` commands fail are fairly clear in explaining what's missing, but I figure if we mention these up front it might save the reader some debug cycles.